src/components/Authentication/LoginForm.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { useContext } from 'react';
 import { Alert, Spinner } from 'reactstrap';
 import baseUrl from '../../utils/baseUrl';
-import { EscolaLMSContext } from '@escolalms/connector/lib/context';
+import { EscolaLMSContext } from '@escolalms/sdk/lib/context';
```

Comment:
what do you think about storing react stuff in one folder 
example 
`@escolalms/sdk/lib/react/context'